jobs/future-effective-filings/file_future_effective.py

```diff
@@ -108,11 +100,12 @@ async def run(loop, application: Flask = None):  # pylint: disable=redefined-out
                     filer_topic = current_app.config.get("ENTITY_FILER_TOPIC", "filer")
                     queue.publish(topic=filer_topic, payload=queue.to_queue_message(cloud_event))
                     structured_log(request, "INFO", f"publish to filer for id: {filing_id}")
-                    application.logger.debug(f'Successfully put filing {filing_id} on the queue.')
-        except Exception as err:  # pylint: disable=broad-except
-            application.logger.error(err)
+                    application.logger.debug("Successfully put filing %s on the queue.", filing_id)
```

Comment:
Not a huge difference, but any particular reason for the switch from the f-string format (`f"Successfully put filing {filing_id}..."`) to using the `%` operator (`"Successfully put filing %s..."`)?

Reply:
> Not a huge difference, but any particular reason for the switch from the f-string format (`f"Successfully put filing {filing_id}..."`) to using the `%` operator (`"Successfully put filing %s..."`)?
> 
> Edit: or was this done automatically by the formatter?

https://pylint.readthedocs.io/en/stable/user_guide/messages/warning/logging-fstring-interpolation.html
It comes from `W1203 logging-fstring-interpolation` warning from `pylint`